queries.py

```diff
@@ -87,3 +88,52 @@ def get_expenditure_by_country_func_econ_year():
     """
     df = execute_query(query)
     return df
+
+
+def get_basic_country_data(countries):
+    country_list = "', '".join(countries)
+    query = """
+        SELECT * FROM indicator.country
+    """
+    query += f" WHERE country_name IN ('{country_list}')"
+    df = execute_query(query)
+    return df
+
+
+def get_expenditure_by_country_geo1_year():
+    query = """
+        SELECT * FROM boost.expenditure_by_country_geo1_year
+    """
+    df = execute_query(query)
+    return df
+
+
+def get_adm_boundaries(countries):
+    query = """
+        SELECT * FROM indicator.admin1_boundaries_gold
+    """
+    country_list = "', '".join(countries)
+    query += f" WHERE country_name IN ('{country_list}')"
+    query += " ORDER BY country_name"
+    df = execute_query(query)
+    geojson = {
```

Comment:
let's just do query execution in this function.
we can do the processing of the queried data outside.